com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -2076,11 +2070,11 @@ DepthOfFieldParameters PrepareDoFParameters(HDCamera camera)
 
             parameters.threadGroup8 = new Vector2Int(threadGroup8X, threadGroup8Y);
 
-            parameters.physicalCameraCurvature = m_PhysicalCamera.curvature;
-            parameters.physicalCameraAnamorphism = m_PhysicalCamera.anamorphism;
-            parameters.physicalCameraAperture = m_PhysicalCamera.aperture;
-            parameters.physicalCameraBarrelClipping = m_PhysicalCamera.barrelClipping;
-            parameters.physicalCameraBladeCount = m_PhysicalCamera.bladeCount;
+            parameters.physicalCameraCurvature = camera.camera.curvature;
+            parameters.physicalCameraAnamorphism = camera.camera.anamorphism;
+            parameters.physicalCameraAperture = camera.camera.aperture;
+            parameters.physicalCameraBarrelClipping = camera.camera.barrelClipping;
+            parameters.physicalCameraBladeCount = camera.camera.bladeCount;
```

Comment:
Maybe you could introduce a local variable, `physicalCamera`

Reply:
Fixed